tests/unit/cluster/multidb.tcl

```diff
@@ -0,0 +1,401 @@
+# Tests multi-databases in cluster mode
+
+# source "../tests/includes/init-tests.tcl"
```

Comment:
```suggestion
```
Presumably this is some left over code from when the tests were in the old framework?